debian/tests/integrationtests

```diff
@@ -4,7 +4,16 @@ set -e
 
 echo 'ubuntu ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers.d/autopkgtest
 
-su ubuntu -c "SNAPCRAFT_FROM_INSTALLED=1 TEST_STORE=fake ADT_TEST=1 python3 -m unittest discover -b -v -s integration_tests"
-su ubuntu -c "SNAPCRAFT_FROM_INSTALLED=1 TEST_STORE=fake ADT_TEST=1 python3 -m unittest discover -b -v -s integration_tests/store"
-su ubuntu -c "SNAPCRAFT_FROM_INSTALLED=1 TEST_STORE=fake ADT_TEST=1 python3 -m unittest discover -b -v -s integration_tests/plugins"
-su ubuntu -c "SNAPCRAFT_FROM_INSTALLED=1 SNAPCRAFT_SLOW_TESTS=1 TEST_STORE=fake ADT_TEST=1 PATH=/snap/bin:$PATH python3 -m unittest discover -b -v -s integration_tests/snapd"
+if [ -z "$SNAPCRAFT_AUTOPKGTEST_SUITES" ]; then
+    suites="integration_tests integration_tests/store integration_tests/plugins integration_tests/snapd"
```

Comment:
Since you're running a `discover` below, are you sure you want the parent dir (`integration_tests`) included here?

Reply:
Scratch this comment-- these dirs aren't packages (no `__init__.py`.